app.yaml

```diff
@@ -32,6 +32,7 @@ handlers:
 - url: /build
   static_dir: build
   secure: always
+  application_readable: true
```

Comment:
See other comment -- I wonder if we can do gadgets/etc. in a separate pass, given that there seems to be some additional work needed before we can do it "properly".